cmd/infra/powervs/service_id.go

```diff
@@ -0,0 +1,153 @@
+package powervs
+
+import (
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"strings"
+
+	credreqv1 "github.com/openshift/cloud-credential-operator/pkg/apis/cloudcredential/v1"
+	cco "github.com/openshift/cloud-credential-operator/pkg/cmd/provisioning/ibmcloud"
+	ccoibmcloud "github.com/openshift/cloud-credential-operator/pkg/ibmcloud"
+)
+
+var kubeCloudControllerManagerCR = `
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-powervs-cloud-controller-manager
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: IBMCloudPowerVSProviderSpec
+    policies:
+    - attributes:
+      - name: resourceType
+        value: resource-group
+      roles:
+      - crn:v1:bluemix:public:iam::::role:Viewer
+    - attributes:
+      - name: serviceName
+        value: is
+      roles:
+      - crn:v1:bluemix:public:iam::::role:Editor
+      - crn:v1:bluemix:public:iam::::role:Operator
+      - crn:v1:bluemix:public:iam::::role:Viewer
+    - attributes:
+      - name: serviceName
+        value: power-iaas
+      roles:
+      - crn:v1:bluemix:public:iam::::role:Viewer
+      - crn:v1:bluemix:public:iam::::serviceRole:Reader
+      - crn:v1:bluemix:public:iam::::serviceRole:Manager
+`
+var nodePoolManagementCR = `
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-capi-powervs
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: IBMCloudPowerVSProviderSpec
+    policies:
+    - attributes:
+      - name: serviceName
+        value: power-iaas
+      roles:
+      - crn:v1:bluemix:public:iam::::serviceRole:Manager
+      - crn:v1:bluemix:public:iam::::role:Editor
+`
+
+var ingressOperatorCR = `
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-ingress-powervs
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: IBMCloudPowerVSProviderSpec
+    policies:
+    - attributes:
+      - name: serviceName
+        value: internet-svcs
+      roles:
+      - crn:v1:bluemix:public:iam::::serviceRole:Manager
+      - crn:v1:bluemix:public:iam::::role:Editor
+`
+
+const (
+	KubeCloudControllerManager = "kubeCloudControllerManager"
+	NodePoolManagement         = "nodePoolManagement"
+	IngressOperator            = "ingressOperator"
+)
+
+var SecretsToManage = map[string]string{
```

Comment:
no need to export this var if not used outside the package

Reply:
not using this anymore, since map logic is removed.